tools/runqslower.py

```diff
@@ -42,14 +42,17 @@
 examples = """examples:
     ./runqslower         # trace run queue latency higher than 10000 us (default)
     ./runqslower 1000    # trace run queue latency higher than 1000 us
-    ./runqslower -p 123  # trace pid 123 only
+    ./runqslower -p 123  # trace pid 123 and its children threads
```

Comment:
just say 'trace pid 123'. people should understand what it means based linux user space convention.